lib/helper.js

```diff
@@ -44,7 +45,7 @@ var validationType = function(value, validation, key){
       return {value: value};
     }else{
       if(types[validation].type) validation = types[validation].type;
-      return {error: key + ': ' + value + ', has to be ' + validation + ' type.', param: key};
+      return {error: key + ': ' + escapeHtml(value) + ', has to be ' + validation + ' type.', param: key};
```

Comment:
what is the HTML template these are ultimately rendered in, and can we do the escaping there?